sites.json

```diff
@@ -155,6 +155,15 @@
     "categories": [
       "docs",
       "blog"
+    ],
+  {
```

Comment:
There appears to be a missing closing brace here.

Reply:
TY I see where I got confused.